modules/plot/main/coffee/Sparkline.coffee

```diff
@@ -37,7 +37,7 @@ class Sparkline
     hx.components.clear(selector)
     hx.component.register(selector, this)
 
-    graph = new hx.Graph(selector)
+    graph = new hx.Graph(selector, options)
```

Comment:
I just found it weird that the Sparkline has no control over the Graph it is rendering at all... Why is Sparkline not extending Graph but just using it in the background anyway?